packages/shared/src/hooks/usePostById.ts

```diff
@@ -22,24 +24,31 @@ const usePostById = ({
   options = {},
 }: UsePostByIdProps): UsePostById => {
   const { tokenRefreshed } = useAuthContext();
+  const isEnabled = !!id && tokenRefreshed;
   const {
     data: postById,
     isLoading,
     isFetched,
   } = useQuery<PostData>(
     ['post', id],
     () => request(graphqlUrl, POST_BY_ID_QUERY, { id }),
-    { ...options, enabled: !!id && tokenRefreshed },
+    {
+      ...options,
+      enabled: isNullOrUndefined(options.enabled)
+        ? isEnabled
+        : isEnabled && options.enabled,
```

Comment:
I think we can leave it as it was right?

Reply:
Yes, was just thinking in the future, but let's remove it as it is not too complex to add in the future.